operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

```diff
@@ -417,18 +430,24 @@ Verify that the Lustre file system is available from the management cluster.
     1. Check the status again.
 
         ```bash
-        ncn-m001# pdsh -w ncn-m00[1-3],ncn-w00[1-3] systemctl status cfs-state-reporter
+        ncn-m001# pdsh -w $(grep -oP 'ncn-\w\d+' /etc/hosts | sort -u |  tr -t '\n' ',') systemctl status cfs-state-reporter
+        ```
+
+    1. If there are still issues with `cfe-state-reporter` on the storage nodes, then it might be due to a spire issue which can be addressed with this script.
```

Comment:
```suggestion
    1. If there are still issues with `cfe-state-reporter` on the storage nodes, then it might be because of a Spire issue which can be addressed with this script.
```